doc-content/enterprise-only/employee-rostering/er-deploy-script-proc.adoc

```diff
@@ -5,13 +5,15 @@ You can deploy the Employee Rostering starter application to {OPENSHIFT} using t
 
 .Prerequisites
 * You are logged in to the target OpenShift environment using the `oc` command line tool. For more information about this tool, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/cli_reference/[CLI Reference].
-* Maven and a Java Development Kit are installed on your local system.
+* OpenJDK {OPTAPLANNER_JDK_VERSION} or later is installed. Red Hat build of Open JDK is available from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required).
+* Apache Maven {OPTAPLANNER_MAVEN_VERSION} or higher is installed. Maven is available from the https://maven.apache.org/[Apache Maven Project] website.
+
 * A `bash` shell environment is available on your local system.
 
 .Procedure
 . Download the `kogito-1.5.0-maven-repository.zip` file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
 . Expand the downloaded archive.
-. Copy the contents of the `kogito-1.5.0.GA-maven-repository/maven-repository` subdirectory into the `~/.m2/repository` directory.
+. Navigate to the `optaweb-8.5.0.Final-redhat-00001/optaweb-employee-rostering` folder.
```

Comment:
@emmurphy1 is this intentional? I'm afraid the procedure stops making sense.

Reply:
@yuloc I can't remember why I made that change, but I had a look and reorganized the section. Would you please confirm? Also, about the file names, would you please scan these drafts and see if I have the right or not:
http://file.ork.redhat.com/~emmurphy/BXMSDOC-7577-dm/
http://file.ork.redhat.com/~emmurphy/BXMSDOC-7577-pam/

Reply:
@emmurphy1 looks good to me now.